setup.py

```diff
@@ -16,6 +16,7 @@
 from build.commands.start import Start
 
 setup_requires = ['Django',
+                  'django-debug-toolbar',
```

Comment:
The webapp is now dependent on the debug toolbar being installed to run. Is it possible to have this as an option for developers, or is it integrated into the code itself?